public/app/features/transformers/editors/ConvertFieldTypeTransformerEditor.tsx

```diff
@@ -31,6 +32,13 @@ export const ConvertFieldTypeTransformerEditor = ({
   onChange,
 }: TransformerUIProps<ConvertFieldTypeTransformerOptions>) => {
   const allTypes = allFieldTypeIconOptions.filter((v) => v.value !== FieldType.trace);
+  const timeZoneOptions: Array<SelectableValue<string>> = [];
+
+  // Format timezone options
+  const tzs = getTimeZones();
```

Comment:
I wonder if we should also set the `includeInternal` flag and add the UTC, for example, as an option. I think it would be nice to have.